Use different git2 fork. See link below for more info.

https://github.com/rust-lang/git2-rs/pull/859
===================================================================
diff --git a/Cargo.toml b/Cargo.toml
index f0eb26c0..4d88f27a 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -68,7 +68,7 @@
 bytesize = { version = "1.1.0" }

 [dependencies.git2]
-version = "0.16"
+git = "https://github.com/Teutates/git2-rs"
 optional = true
 default-features = false

